Styling the User Controls

3Ustin opened the pull request:
The styling of the User Controls was a lot more difficult than we anticipated. The middle slider is to be replaced by the song position slider which will need styling. The middle slider is the volume control and will need to be moved when the song position is done.

3Ustin committed:
Styling the User Controls

The styling of the User Controls was a lot more difficult than we anticipated. The middle slider is to be replaced by the song position slider which will need styling. The middle slider is the volume control and will need to be moved when the song position is done.

MtuMartian commented on public/index.html:
```diff
@@ -24,7 +24,7 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>MusicFree</title>
+    <title>Moosic</title>
```
:cow:

MtuMartian reviewed (approved):
LGTM

minor: I noticed a comment that was probably added during debugging in a previous PR that can be removed

https://github.com/3Ustin/spotify_web_client/blob/main/src/components/UserControls.tsx#L28

3Ustin committed:
update for PR Comments

Tech Lead Miles had left a comment stating I had a comment that was unnecessarily commenting.
